### DISCUSSION TITLE
Feature Request: Integration with DocuSeal for Signature Workflow in Paperless-ngx

### DISCUSSION THREAD

**Discussion by axgd-code** [docusealco/docuseal#91] [Ideas]
Hi there,

As an enthusiastic user of DocuSeal, I'd like to propose the development of an API that would allow seamless integration of DocuSeal's features into third-party applications (like Paperless-ngx). Specifically, I believe that having an API for DocuSeal would greatly benefit users in the following ways:

As a logged-in user of Paperless-ngx for example:
- I would like to be able to call DocuSeal to initiate a signature workflow.
- I wish to be able to track the progress of the signature workflow (number of signatories).
- I would appreciate the ability to view the deadline date within Paperless-ngx.

Thank you in advance for your feedback.

**Comment by AlexBTurchyn** [comment]
@axgdcode Can you please provide a detailed description of the necessary APIs for implementing seamless integration with Paperless-ngx?

**Reply by compgeniuses** [reply]
I think this is something i can also contribute in its documentation as well

**Comment by compgeniuses** [comment]
An intergation between Paperless NGX with docuseal would be an amazing functionality to have.

the files hosted in paperless can be signed with docuseal,

and files to be signed by docuseal, could be sent to paperless for hosting
